packages/ariakit/src/combobox/__examples__/combobox-cancel/list.ts

```diff
@@ -0,0 +1,9 @@
+export default [
+  "Apple",
+  "Bacon",
+  "Banana",
+  "Broccoli",
+  "Burger",
+  "Cake",
+  "Candy",
+];
```

Comment:
We can delete this file and move the list to the index file.

Reply:
Yes I was thinking about that, no need to have such a small list in a file